test/LearningTest/TestCoclustering/Standard/DeployDistributionAdult2varsFrequency/test.prm

```diff
@@ -0,0 +1,67 @@
+// Fri Dec 02 15:33:16 2011
+// c:\Users\boullema\Learning\Win32\Debug\MODL_Coclustering.exe
+// Output command file
+//
+//This file contains recorded commands, that can be replayed.
+//Commands are based on user interactions:
+//	field update
+//	list item selection
+//	menu action
+//Every command can be commented, using //.
+//For example, commenting the last Exit command will allow other
+//user interactions, after the commands have been replayed.
+//
+//

```

Comment:
@marcboulle  we should eventually get rid of local paths in the code. Furthermore, could we get rid of this boilerplate, maybe with an option ?